### DISCUSSION TITLE
pool error

### DISCUSSION THREAD

**Discussion by benjaminwnelson** [amices/mice#340] [Impute, analyse and pool]
When I run summary(mice::pool.fitlist(model)) it gives this error. Any on how to address this would be appreciated!

Error in h(simpleError(msg, call)) : 
  error in evaluating the argument 'object' in selecting a method for function 'summary': No tidy method for objects of class qr
In addition: Warning messages:
1: In get.dfcom(object, dfcom) : Infinite sample size assumed.
2: 'tidy.numeric' is deprecated.
See help("Deprecated")

**Comment by stefvanbuuren** [comment]
Please search the closed issues before opening a new one.

**Comment by stefvanbuuren** [comment]
Related: #106, #123, #217
